disnake/onboarding.py

```diff
@@ -186,11 +206,41 @@ def __repr__(self) -> str:
         )
 
     @property
-    def roles(self) -> List[Role]:
-        """List[:class:`Role`]: A list of roles that will be added to the user when they select this option."""
-        return list(filter(None, map(self.guild.get_role, self.role_ids)))
+    def id(self) -> int:
+        """:class:`int`: The prompt option's ID."""
+        return self._id
```

Comment:
Similar to the other one, this can also be a simple attribute as far as I can tell.

Reply:
oops, I forgot to revert this one

Reply:
d1b46aad4b38287220440969799b331fa8091c2d my bad haha